app/controllers/concerns/authenticated_api_concern.rb

```diff
@@ -1,10 +1,12 @@
 module AuthenticatedApiConcern
   extend ActiveSupport::Concern
 
+  HEADER_NAME = "HTTP_GOVUK_ACCOUNT_SESSION".freeze
```

Comment:
I generally like stuff like this broken out - but why now?
Was it just something you spotted on the way past?

Reply:
Oh cool, [there's my answer](https://github.com/alphagov/account-api/pull/261/commits/6400a2ebcd34b53bafbaa4b5d8522ad1620b872e#diff-7e6894647a98a35c83ab744086a6f542f6022c2837ec1ca905ce629ec22cd5c3R22)